raiden-dapp/src/components/AmountDisplay.vue

```diff
@@ -24,10 +24,16 @@ import type { Token } from '@/model/types';
 export default class AmountDisplay extends Vue {
   @Prop({ required: false, default: false, type: Boolean })
   exactAmount!: boolean;
+
   @Prop({ required: true })
   amount!: string | BigNumber;
+
+  @Prop({ type: String, default: '' })
```

Comment:
I don't think you need to have that empty string default value?

Reply:
I do. Else the rendering will throw an error as we do not render it condiationally.